### PR TITLE
archive explorer blogpost draft

### PR DESCRIPTION
**Pull request by 93Boy**

### CODE REVIEW

**Comment by stschiff** [comment]
OK, extremely good English and clear structure, did you use AI to help you here? 

For my taste it's a bit too salesman-like, and the term "Poseidon Archive Explorer" is repeated way too often. 

Also, I would suggest to make it more a personal story. What is your own route to this data? You have worked on it for quite a while, you have curated quite a bit of data yourself and know how much work it is. Now this is the first step to publicise it and make it more accessible. Something like this.

**Comment by 93Boy** [comment]
The content I have written myself. but I have used tools like Grammarly to fix grammar mistakes, substitute more proficient technical words in a few places, and adjust sentence structures. I understand your point. Initially, I thought this post should sound more like an announcement. I will rewrite it accordingly

File: Archive_explorer_blogpost.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+In the ever-evolving world of ancient DNA research, staying up-to-date with the latest publications and datasets is essential. To address the challenges faced by researchers in this field, the Poseidon ecosystem has introduced a new feature - the Poseidon Archive Explorer. This dynamic web application offers a comprehensive overview of current Ancient DNA publications, providing a valuable resource for scholars and enthusiasts alike.
+
+In the Poseidon ecosystem, there are three main components: The Poseidon Community Archive, The Poseidon Minotaur Archive, and The Poseidon AADR Archive. These repositories are carefully organized to store valuable ancient DNA data. To learn more about each of their specific roles, you can visit this link [https://www.poseidon-adna.org/#/archive_overview].The Poseidon Ecosystem further underscores its commitment to data integrity by integrating an in-house web API with the GitHub Large File Storage system. This hybrid approach ensures robust version control and data cleanliness. Researchers and enthusiasts can benefit from this combination, fostering a harmonious ecosystem of data accessibility and precision.
```
Comment (stschiff) [inline Archive_explorer_blogpost.txt]: Well, I would say there are 1) The Poseidon Package format (provide link to doc), 2) a set of tools to work with packages (provide links to trident and xerxes), and 3) public repositories, of which there are three: ...

Comment (stschiff) [reply Archive_explorer_blogpost.txt]: Also I would replace "valuable Ancient DNA data" with "Ancient DNA genotyping data together with archaeological and laboratory context information".

File: Archive_explorer_blogpost.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The Poseidon Archive Explorer: Bridging the Gap
```
Comment (stschiff) [inline Archive_explorer_blogpost.txt]: If this is supposed to be a heading, just say "Bridging the Gap".

File: Archive_explorer_blogpost.txt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+The Poseidon Archive Explorer was developed with the aim of addressing the technical challenges often encountered by users when navigating the Poseidon ecosystem. This innovative web application provides a user-friendly interface that simplifies the exploration of the extensive ancient DNA data. Here are some of its prominent features:
```
Comment (stschiff) [inline Archive_explorer_blogpost.txt]: remove "innovative", in my view. We are not the first with such an app.

File: Archive_explorer_blogpost.txt [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+                                        *)Dual Archive Support: The Poseidon Archive Explorer seamlessly supports both The Poseidon Community Archive and The Poseidon AADR Archive. This integration provides users with access to a diverse range of ancient DNA publications.
```
Comment (stschiff) [inline Archive_explorer_blogpost.txt]: Here, some screenshots would be great!

File: Archive_explorer_blogpost.txt [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+                                        *)Real-Time Visualization: A key highlight of the Archive Explorer is its ability to real-time visualize the current available samples on a world map. This interactive feature allows users to geographically locate and explore datasets with ease.
+
+                                        *)Effortless Data Downloads: Researchers often require access to genotype data for their studies. The Archive Explorer simplifies this process with a "One-click download" button, streamlining the retrieval of essential genotype information.
+
+The Archive Explorer: A Bridge Between Data and You
```
Comment (stschiff) [inline Archive_explorer_blogpost.txt]: again, just the second part of the heading, I suggest, to remove all too many mentions of "Archive Explorer"

File: Archive_explorer_blogpost.txt [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+The concept behind the development of Archive Explorer is pretty straightforward. We've been housing all our Poseidon genotype data using the GIT Large File Storage system (GIT LFS). While that's all fine and dandy for version control, we realized we needed something more interactive and user-friendly to share and visualize this data with the public. The solution? A dynamic web application built on JavaScript.
```
Comment (stschiff) [inline Archive_explorer_blogpost.txt]: The truth is a bit more complicated. Git LFS is not even the main backend for the app. The server is separate from Git LFS. I can formulate something more accurate later, when my other points have been addressed.

File: Archive_explorer_blogpost.txt [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+Navigating the Archive Explorer:
+
+When you land on the Archive Explorer page, you'll notice it's got some neat features. Up top, there's an archive selector. This is your gateway to choose the specific archive you're interested in. Then, we've thrown in a world map powered by the Leaflet map plugin because it's open source and super easy to use.
```
Comment (stschiff) [inline Archive_explorer_blogpost.txt]: Cool, screenshots would be nice, perhaps.

File: Archive_explorer_blogpost.txt [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+1) If you know exactly what you're looking for, you can type a full or partial name in our dynamic search bar and select the package you want.
+
+2) Want to explore data from a specific part of the world? Check out the world map, click on a marker, and voila! You'll see a snapshot of what's in that package, and a link to dive straight into it.
+
+3) For those who prefer the manual approach, we've got a table view of all the publications available in each archive. You can roll up your sleeves and search for the info you need.
+
+Getting Up Close and Personal with the Data
+
+Once you've picked a package, you can access it either through the map marker or by hitting that trusty magnifying glass icon on the table view. If you're into details, the table view lets you view genotype data in GIT LFS or download it as a "zip" file.
```
Comment (stschiff) [inline Archive_explorer_blogpost.txt]: No, it's not git LFS, but our Poseidon Server, which is separate from Git.

**Review by stschiff — CHANGES_REQUESTED** [review]
Nice, @93Boy, well done. Please bring it into Markdown format and add it to the posts directory to be included in the rendered webpage. You should also think of images. Usually the first image of the post is used as a thumbnail for the Post-list, but that can also be tweaked. You can also add screenshots here and there to spice it up. But I'll leave that to you. But at least one image would be nice, so that the renderer has something to add to the article on the blog list page.

**Comment by stschiff** [comment]
Hi @93Boy. Thanks. Could you please respond to my comments and either flag them as resolved or comment back? Also, as mentioned, the format for the blog needs to be markdown. Please see this [Markdown help page](https://quarto.org/docs/authoring/markdown-basics.html) from quarto for details. It's quite easy. Just just have to markup headings, more or less.

**Comment by stschiff** [comment]
Thanks again for the markdown format change! Please remove the old `.txt` and `.tex` files. And please reply to my comments above (or just feel free to resolve them). I'll then take it from there and merge.

File: Archive_explorer_blogpost.txt (hunk 1 above)
Comment (93Boy) [reply Archive_explorer_blogpost.txt]: These changes were fixed in latest revision

File: Archive_explorer_blogpost.txt (hunk 2 above)
Comment (93Boy) [reply Archive_explorer_blogpost.txt]: Changed as requested

File: Archive_explorer_blogpost.txt (hunk 3 above)
Comment (93Boy) [reply Archive_explorer_blogpost.txt]: Changed as requested

File: Archive_explorer_blogpost.txt (hunk 5 above)
Comment (93Boy) [reply Archive_explorer_blogpost.txt]: I have removed some of the repetitions

**Comment by stschiff** [comment]
What's the status here, @93Boy? Shall I review again and merge if possible?

**Comment by 93Boy** [comment]
Yes @stschiff I have corrected all the points you have mentioned and added some screenshots as well. Can you kindly review the document?

**Comment by stschiff** [comment]
Yes, thanks, I'm on it.

**Pull request merged**